react/ActionMenu/Readme.md

```diff
@@ -109,11 +109,9 @@ const anchorRef = React.createRef();
 </div>
 ```
 
-### ActionMenu within other components and containerElRef
+### ActionMenu within modals
```

Comment:
I think we can remove this section, `ActionMenu inside custom fixed elements` is enough imo.

Reply:
As discussed, I kept the section to benefit from Argos tests, and put the modal section below (for reading coherence, you learn about the technique, then learn about its usage in Modals).